crates/subspace-farmer/README.md

```diff
@@ -37,6 +37,8 @@ RocksDB on Linux needs LLVM/Clang:
 sudo apt-get install llvm clang
 ```
 
+For OpenCL support before enabling `opencl` feature make sure to follow documentation of `sloth256-189` library: <https://docs.rs/sloth256-189/latest/sloth256_189/opencl/index.html>.
```

Comment:
Maybe we can point to exact version of opencl?
https://docs.rs/sloth256-189/0.4.2/sloth256_189/opencl/index.html

Reply:
And in other places